br/br-auto-tune.md

```diff
@@ -0,0 +1,89 @@
+---
+title: BR Auto-Tune
+summary:
+---
+
+# BR Auto-Tune <span class="version-mark">New in v5.4.0</span>
+
+Before TiDB v5.4.0, when you back up data using BR, the number of threads used for backup makes up 75% of the logical CPU cores. Without a speed limit, the backup process can consume a lot of cluster resources, which has a considerable impact on the cluster performance. Although you can reduce the impact of backup by adjusting the size of the thread pool, it is a tedious task to observe the CPU load and manually adjust the thread pool size.
+
+To reduce the impact of backup tasks on the cluster, starting from TiDB v5.4.0, BR introduces the auto-tune feature. When the cluster resource utilization is high, BR automatically limits the resources used by backup tasks and thereby reduces the impact on the cluster. The auto-tune feature is enabled by default.
+
+## User scenario
+
+If you hope to reduce the impact of backup tasks on the cluster, you can enable the auto-tune feature. With this feature enabled, BR performs backup tasks as fast as possible without excessively affecting the cluster.
+
+Alternatively, you can limit the backup speed by using the TiKV configuration item [`backup.num-threads`](/tikv-configuration-file.md#num-threads-1) or using the parameter `--ratelimit`.
+
+## Use auto-tune
+
+The auto-tune feature is enabled by default, without additional configuration.
+
+> **Note:**
+>
+> For clusters that upgrade from versions earlier than v5.4.0 to v5.4.0 or later versions, the auto-tune feature is disabled by default. You need to manually enable it.
```

Comment:
In fact, only cluster update from v5.3.0 would disable this by default.

(I just noticed that the same issue also applies to Chinese document...😢)

Reply:
What's the behavior of clusters < v5.3.0?
And how about clusters of v5.3.x (e.g. v5.3.1)?